core/services/keeper/registry_synchronizer_core.go

```diff
@@ -93,13 +93,7 @@ func (rs *RegistrySynchronizer) Start(context.Context) error {
 				keeper_registry_wrapper.KeeperRegistryConfigSet{}.Topic():        nil,
 				keeper_registry_wrapper.KeeperRegistryUpkeepCanceled{}.Topic():   nil,
 				keeper_registry_wrapper.KeeperRegistryUpkeepRegistered{}.Topic(): nil,
-				keeper_registry_wrapper.KeeperRegistryUpkeepPerformed{}.Topic(): {
-					{},
-					{},
-					{
-						log.Topic(rs.job.KeeperSpec.FromAddress.Hash()),
```

Comment:
Race conditions might happen due to this case. We need to document it somewhere since it was discussed many times lol
![IMG_5459](https://user-images.githubusercontent.com/13855864/160901702-0a1c1a74-dda0-41ae-9d5a-763ef676c398.jpg)

Reply:
thanks I updated the query to handle this

Reply:
thanks I updated the query to handle this